src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax.service;
 
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws RegistrationException;
```

Comment:
You shouldn't made this change, please restore it to its previous form.

Reply:
ok, done